config/owasp/suppressions.xml

```diff
@@ -44,4 +44,10 @@
     <gav regex="true">^org\.springframework\.security:.*$</gav>
     <cve>CVE-2018-1258</cve>
   </suppress>
+  <!-- Temporary suppression until dependencies are upgraded -->
```

Comment:
Vulnerability is even in current version just older version of scanner did not spot it. Will be removed in next PR that will cover update of all app dependencies.